pkg/kubectl/genericclioptions/builder_flags.go

```diff
@@ -14,68 +14,95 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package wait
+package genericclioptions
 
 import (
-	"strings"
-
-	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
-	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
 	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/resource"
 )
 
 // ResourceBuilderFlags are flags for finding resources
 type ResourceBuilderFlags struct {
-	FilenameOptions resource.FilenameOptions
+	FileNameFlags *FileNameFlags
 
-	LabelSelector     string
-	FieldSelector     string
-	AllNamespaces     bool
 	Namespace         string
 	ExplicitNamespace bool
 
-	// TODO add conditional support.  These are false for now.
-	All   bool
-	Local bool
+	LabelSelector *string
+	FieldSelector *string
+	AllNamespaces *bool
+
+	All   *bool
```

Comment:
used or dead?  If dead.  Wire up in a followup

Reply:
Used in the `ToBuilder` method as an argument to the `ResourceTypeOrNameArgs` param